modules/fedimint-lnv2-tests/src/bin/tests.rs

```diff
@@ -142,72 +207,88 @@ async fn test_self_payment_success(dev_fed: &DevJitFed) -> anyhow::Result<()> {
         (gw_cln.addr.clone(), gw_lnd.addr.clone()),
         (gw_cln.addr.clone(), gw_cln.addr.clone()),
     ] {
-        let (invoice, receive_op) = fetch_invoice(&client, &gw_receive, 1_000_000).await?;
+        info!("Testing self payment success: {gw_send} -> {gw_receive}");
+
+        let (invoice, receive_op) = receive(&client, &gw_receive, 1_000_000).await?;
 
         test_send(&client, &gw_send, &invoice, FinalSendState::Success).await?;
 
         await_receive_claimed(&client, receive_op).await?;
     }
 
-    info!("test_self_payment successful");
+    info!("Testing self payments success successful");
 
     Ok(())
 }
 
-async fn test_lightning_payment(dev_fed: &DevJitFed) -> anyhow::Result<()> {
-    let fed = dev_fed.fed().await?;
-    let client = fed.new_joined_client("lnv2-lightning-test-client").await?;
+async fn test_lightning_payments(dev_fed: &DevJitFed) -> anyhow::Result<()> {
+    info!("Testing lightning payments...");
+
+    let federation = dev_fed.fed().await?;
+
+    let client = federation
+        .new_joined_client("lnv2-lightning-payments-client")
+        .await?;
+
     let gw_lnd = dev_fed.gw_lnd().await?;
     let gw_cln = dev_fed.gw_cln().await?;
+
     let lnd = dev_fed.lnd().await?;
     let cln = dev_fed.cln().await?;
 
-    // Verify HOLD invoices still work, create one now for payment later
+    info!("Verify HOLD invoices still work, create one now for payment later");
```

Comment:
Nit: The other logs here are all present tense ("Testing"), can we change this to "Verifying HOLD invoices..."

Reply:
Meh, actually I think it's fine the way it is